tests/unit/test_server.py

```
@@ -0,0 +1,315 @@
+    orch.delete = AsyncMock(return_value=True)
+    orch.prune = AsyncMock(return_value=5)
+    orch.stats = AsyncMock(
+        return_value={"active_count": 10, "archived_count": 2, "total_count": 12, "project": None}
+    )
+    orch.dispose = AsyncMock(return_value=None)
+    return orch
+
+
+@pytest.fixture
+def chronos_server(mock_orchestrator: MagicMock):
+    """ChronosServer インスタンスを返す（初期化済み状態に設定）。"""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)